src/components/Table/TableDetailWizard/TableDetailWizard.jsx

```diff
@@ -45,9 +46,9 @@ const StyledFooter = styled.div`
 `;
 
 const StyledContentContainer = styled.div`
-  padding-top: 30px;
-  padding-right: 50px;
-  padding-left: 40px;
+  padding-top: ${spacing07};
```

Comment:
Does not map to 30px. Have we checked what the ui implications are here?

Reply:
This will not break the layout so it can be standardized 👍